feat(guide): Starr language tips add German dual and multi (#1880)
TRaSH-Guides/Guides: docs/Radarr/Tips/How-to-setup-language-custom-formats.md

    [[% filter indent(width=4) %]][[% include 'json/guide-only/language-not-original.json' %]][[% endfilter %]]
    ```

> !!! tip

Reviewer: ```suggestion
!!! tip

```

    [[% filter indent(width=4) %]][[% include 'json/guide-only/language-not-english.json' %]][[% endfilter %]]
    ```

> !!! tip

Reviewer: ```suggestion
!!! tip

```

    [[% filter indent(width=4) %]][[% include 'json/guide-only/language-prefer-german.json' %]][[% endfilter %]]
    ```

> !!! tip

Reviewer: ```suggestion
!!! tip

```

    [[% filter indent(width=4) %]][[% include 'json/guide-only/language-prefer-german.json' %]][[% endfilter %]]
    ```

> !!! tip

Reviewer: ```suggestion
!!! tip

```

---

## Language Special Cases

Here we will add special cases often related to a specific language.
example: German Dual Language and Multi-language releases

> !!! info "In general Dual/Multi stands for original language (*movies/show original language*) + foreign language(s), which audio track is used as first track isn't always known."

Reviewer 2: ```suggestion
Here we will add special cases that often are related to specific languages.
example: German Dual Language and/or Multi-language releases

!!! info "In general, Dual/Multi stands for original language (*movies/show original language*) + foreign language(s). Which audio track being used as the first track isn't always known based on the naming."

### Language: German + Original

> This Custom Format can be used in several ways,

Reviewer 2: ```suggestion
This Custom Format can be used in several ways:

> ## Language Special Cases
> 
> Here we will add special cases that often are related to specific languages.
> example: German Dual Language and/or Multi-language releases
> 
> !!! info "In general, Dual/Multi stands for original language (*movies/show original language*) + foreign language(s). Which audio track being used as the first track isn't always known based on the naming."
> 
> ### Language: German + Original
> 
> This Custom Format can be used in several ways:

Reviewer 2: ```suggestion
## Language Special Cases

Here we will add special cases that often are related to specific languages.
example: German Dual Language and/or Multi-language releases

!!! info "In general, Dual/Multi in a release title stands for original language (*movies/show original language*) + foreign language(s). Which audio track is used for the first track isn't always known based solely on the naming."

### Language: German + Original

This Custom Format can be used in two ways:
```

- If you **DON'T WANT** that your media file has a German audio track included add this Custom Format with a score of `-10000`.

> - If you **PREFER** that your media file has a German audio track included then you can add this Custom Format with a positive score.

Reviewer 2: ```suggestion
- If you **DON'T WANT** your media file to have a German audio track included, add this Custom Format with a score of `-10000`.
- If you **PREFER** your media file to have a German audio track included, add this Custom Format with a positive score.
